service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadCMOAboutToStartControllerTest.java

```diff
@@ -33,75 +33,6 @@ protected UploadCMOAboutToStartControllerTest() {
         super("upload-cmo");
     }
 
-    @Test
-    void shouldReturnMultiHearingData() {
-        givenLegacyFlow();
```

Comment:
i guess all test with givenLegacyFlow could be removed, but can be done as part of feature toggle cleanup story